02-peer-protocol.md

```diff
@@ -787,9 +787,20 @@ is destined, is described in [BOLT #4](04-onion-routing.md).
 #### Requirements
 
 A sending node:
-  - MUST NOT offer `amount_msat` it cannot pay for in the
-remote commitment transaction at the current `feerate_per_kw` (see "Updating
-Fees") while maintaining its channel reserve.
+  - if it is _responsible_ for paying the Bitcoin fee:
+    - MUST NOT offer `amount_msat` if, after adding that HTLC to its commitment
+    transaction, it cannot pay the fee for either the local or remote commitment
+    transaction at the current `feerate_per_kw` while maintaining its channel
+    reserve (see "Updating Fees").
```

Comment:
Add note that this is not entirely preventable in certain scenarios (concurrency)?

Reply:
I think this is already explained in the "Updating Fees" section that's linked.
Do you think it's worth replicating this information here?

Reply:
Ok, no then I think it is good as is. Could make "Updating Fees" a link.

Reply:
True, that would be helpful. I'll make this a link to the relevant section.

Reply:
Done in f5490f17d17ff49dc26ee459432b3c9db4fda8a9